src/views/List.jsx

```diff
@@ -53,42 +53,65 @@ export function List({ data, listToken, isLoading }) {
 		}
 	};
 	return (
-		<>
+		<div className="min-h-[100vh]">
 			{isLoading ? (
 				'Loading...'
 			) : data.length ? (
 				<>
 					{listToken && (
-						<div>
+						<div className="flex items-center justify-center gap-2 lg:gap-4 text-[15px] lg:text-[22px] my-3">
 							<p>Your list : {listToken}</p>
-							<button onClick={copyToken}>{copy}</button>
+							<button
+								className="outline-2 p-2 bg-[#C2410C] rounded-lg"
+								onClick={copyToken}
```

Comment:
I think we should add text-white to the copy token button, it would make the text easier to read.